sites/docs/src/content/docs/usage/Getting_started/configuration.md

```diff
@@ -182,12 +237,12 @@ process {
 }
 ```
 
-You can be more specific than this by targeting a given process name instead of it's label using `withName`. You can see the process names in your console log when the pipeline is running For example:
+You can also be more specific than this by targeting a given _process_ (job) name instead of it's label using `withName`. You can see the process names in your console log when the pipeline is running For example:
```

Comment:
```suggestion
You can also be more specific than this by targeting a given _process_ (job) name instead of its label using `withName`. You can see the process names in your console log when the pipeline is running For example:
```